Warning in USM array PYX compilation.

The following unused variable warning is popping up after recent changes:
```
dpctl/tensor/_usmarray.cpp:2528:36: warning: ‘__pyx_f_5dpctl_19_sycl_queue_manager_get_current_queue’ defined but not used [-Wunused-variable]
 2528 | static struct PySyclQueueObject *(*__pyx_f_5dpctl_19_sycl_queue_manager_get_current_queue)(int __pyx_skip_dispatch); /*proto*/
```

--- a/dpctl/_sycl_queue_manager.pxd
+++ b/dpctl/_sycl_queue_manager.pxd
@@ -20,6 +20,6 @@
 from ._sycl_queue cimport SyclQueue
 
 
-cpdef public SyclQueue get_current_queue()
+cpdef SyclQueue get_current_queue()
 cpdef get_current_device_type ()
 cpdef get_current_backend()

